docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Runtime package store | Microsoft Docs
+description: This topic explains the runtime package store and target manifests used by .NET Core.
+keywords: .NET, .NET Core, dotnet store, runtime package store
+author: bleroy
+ms.author: mairaw
+ms.date: 06/26/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 9521d8b4-25fc-412b-a65b-4c975ebf6bfd
+---
+
+# Runtime package store
+
+[!INCLUDE [core-preview-warning](~/includes/core-preview-warning.md)]
+
+Starting with .NET Core 2.0, it's possible to package and deploy apps against a known set of packages that exist in the target environment. The benefits are faster deployments, lower disk space use, and improved startup performance in some cases.
+
+This feature is implemented as a *runtime package store*, which is a directory on disk next to the the [`dotnet` host](../tools/dotnet.md) where packages are stored (typically at *.dotnet/store* in the user's profile). Under this directory, there are subdirectories for architectures and [target frameworks](../../standard/frameworks.md). The file layout is similar to the way that [NuGet assets are laid out on disk](/nuget/create-packages/supporting-multiple-target-frameworks#framework-version-folder-structure):
```

Comment:
Did you create a store? It should appear at *C:\Users\Svick\.dotnet\store* after you create one. Here's a screenshot of mine after working with it ...

![capture](https://user-images.githubusercontent.com/1622880/27772437-4d41f380-5f28-11e7-97bb-6e3868cbc69c.PNG)

Yes, your confusion is warranted. :cry: This is a blown explanation pulled from another source that was referring either to the store located at *C:\Program Files\dotnet\store* or they actually *MOVED* where a custom package store is created. Either way, I'll change this. Thanks for catching it.